helpers/DeliveryHelper.php

```diff
@@ -107,12 +108,12 @@ public static function buildDeliveryData($delivery, $executions)
         }
         
         $deliveryProps = array(
-            new \core_kernel_classes_Property(TAO_DELIVERY_START_PROP),
-            new \core_kernel_classes_Property(TAO_DELIVERY_END_PROP),
+            new \core_kernel_classes_Property(DeliveryContainerService::PROPERTY_START),
```

Comment:
I think that this wrong place for `PROPERTY_START`, it should be here `DeliveryAssemblyService`

Reply:
The constant defined in the DeliveryAssemblyService are constants related to Assembled Delivery based on this fact I have separate them.

Reply:
properties of the delivery should be in the one place
confirmed by @jbout

Reply:
these are properties related to the assembled delivery, they describe the time window during which the tt is allowed to start the test